### PR TITLE
rfe: Added module_utils

### LINKED ISSUE #31: add kubevirt module utility
Many [Ansible modules have utilities](http://docs.ansible.com/ansible/latest/dev_guide/developing_modules.html#appendix-module-utilities) which contain various functions used in modules. I think KubeVirt can have the same, so there will be no duplicated functions across library/kubevirt_*.py

### PR DESCRIPTION
**Pull request by tripledes**
WIP, please don't merge, currently this PR only works for KubeVirt VM module, needs some modifications for OVM and VMRS to work.

* Moved all (most) common methods to module_utils.kubevirt_common.
* Added tests/ansible.cfg for running tests using module_utils directory
so the imports work properly.
* Added kubevirt_helper for functions that don't belong on
KubeVirtAnsibleModule class.
* Added instructions on README.md on how to make ansible to use
tests/ansible.cfg configuration file.
* Modified KubeVirt VM module to use KubeVirtAnsibleModule class.

Fixes #31 #49 #48 #38

### CODE REVIEW

File: module_utils/kubevirt_common.py [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+                return False
+            elif waittime > timeout:
+                return False
+            else:
+                waittime += 5
+                time.sleep(5)
+
+
+    def _build_resource_definition(self, kind):
+        '''return dict() containing the resource definition based of user
```
Comment (alexxa) [inline module_utils/kubevirt_common.py]: s/based of/based on

File: module_utils/kubevirt_common.py [hunk 2]
```diff
@@ -0,0 +1,232 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2018, KubeVirt Team <@kubevirt>
+# Apache License, Version 2.0 (see LICENSE or http://www.apache.org/licenses/LICENSE-2.0)
```
Comment (alexxa) [inline module_utils/kubevirt_common.py]: @tripledes  Sergi, I'm not an expert in Licenses... k8s Ansible functions have "This is a part of Ansible.. " and a reference to GNU license, e.g. see [k8s/common.py](https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/k8s/common.py) However, kubernetes itself has [Apache 2](https://github.com/kubernetes/kubernetes/blob/master/LICENSE). Question: should we put it as "This is a part of Ansible ..." too? 

@fabiand can you comment it please?

**Comment by alexxa** [comment]
@tripledes Sergi, I would make it as 

- module_utils
   |___kubevirt
           |___common.py
           |___helper.py

similar too k8s module_utils  organization: https://github.com/ansible/ansible/tree/devel/lib/ansible/module_utils/k8s

**Comment by tripledes** [comment]
@alexxa could you please approve this PR? Once merged I'll start the agreed rewrite/refactor. Thanks!

**Comment by alexxa** [comment]
@tripledes Please run `ansible-doc kubevirt_{vm,ovm,vmrs}.py -vvvv` on this PR branch. For me  it doesn't work for OVM and VM, but VMRS only.

**Pull request closed without merging**